channeldb/mp_payment.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/lightningnetwork/lnd/lntypes"
 	"github.com/lightningnetwork/lnd/lnutils"
 	"github.com/lightningnetwork/lnd/lnwire"
+	paymentsdb "github.com/lightningnetwork/lnd/payments/db"
```

Comment:
nit: consider name it `pdb` for brevity?

Reply:
currently I prefer the longer name, but happy to change it in PR4, so that I don't end up in rebase hell. (most of the refactor PRs are ready so I would need to rebase everything all the time, so going to rename it after the last refactor for the whole series.